examples/java/src/test/java/org/apache/beam/examples/subprocess/ExampleEchoPipelineTest.java

```diff
@@ -124,7 +122,7 @@
   /**
    * Simple DoFn that echos the element, used as an example of running a C++ library.
    */
-  @SuppressWarnings("serial") @RunWith(JUnit4.class) public static class EchoInputDoFn
```

Comment:
It seems like the @RunWith should be on the outer class.

And if it doesn't work, we should mark it with @Ignore("BEAM-2871")

Reply:
Good call. Added.